src/components/Auth/PasswordForgotForm.tsx

```diff
@@ -52,19 +52,17 @@ const PasswordForgotForm: React.FC = () => {
             type='email'
             required
           />
-          <Button type='submit' fontSize='sm' variant='brand' fontWeight='500' w='100%' h={50}>
+          <Button type='submit' fontSize='sm' variant='primary' fontWeight='500' w='100%' h={50}>
             {t('forgot_password_reset_link')}
           </Button>
         </Flex>
       </FormProvider>
       <Flex flexDirection='column' justifyContent='center' alignItems='start' maxW='100%' mt={0}>
         <Text fontWeight='400' fontSize='sm'>
           {t('already_member')}
-          <NavLink to={Routes.auth.signIn}>
-            <Text as='span' color={'account.link'} ms={1} fontWeight='500'>
-              {t('signin')}
-            </Text>
-          </NavLink>
+          <Link as={NavLink} to={Routes.auth.signIn} ml={1} fontWeight={500}>
+            {t('signin')}
+          </Link>
```

Comment:
IDK why `NavLink` was being used here, but it's a bit cumbersome, since this component adds styles for the currently active path, and it's not required at all for the login button.

Use the normal react-router-dom